docs/pages/deployment/tls-configuration.rst

```diff
@@ -108,6 +108,11 @@ Configuration for `HAProxy <https://www.haproxy.com/>`_ could look like this (gi
 
 Refer to the HAProxy documentation for more information.
 
+.. note::
+
+    In a (level 4) pass-through configuration, the origin IP address for all connections to the Nuts node will be equal to that of the load balancer.
+    You will have to handle IP logging in this configuration yourself.
+
```

Comment:
Can someone please verify this and is this sufficient documentation?

Reply:
I don't think is true, but only applicable to TLS offloading?

Reply:
I think this is true for all configurations. The IP address will be that of the reverse proxy.

The difference between lvl4 and lvl7 is that on lvl 7 you can add the `X-Forwarded-For` header so the node knows the actual origin. AFAIK there is no standard solution for this on lvl4. See [HAProxy docs](https://www.haproxy.com/documentation/hapee/2-4r1/load-balancing/client-ip-preservation/) on this as well.